inference-engine/tests/functional/inference_engine/ir_serialization/models/addmul_abc_execution.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0"?>
+<net name="addmul_abc" version="10">
+	<layers>
+		<layer id="0" name="C" type="Input">
+			<data execOrder="3" execTimeMcs="not_executed" originalLayersNames="C" outputLayouts="x" outputPrecisions="FP32" primitiveType="unknown_FP32" />
+			<output>
+				<port id="0" precision="FP32">
+					<dim>1</dim>
+				</port>
+			</output>
+		</layer>
+		<layer id="1" name="B" type="Input">
+			<data execOrder="1" execTimeMcs="not_executed" originalLayersNames="B" outputLayouts="x" outputPrecisions="FP32" primitiveType="unknown_FP32" />
```

Comment:
@vladimir-paramuzov @dmitry-gorokhov Could you confirm that it is expected format?

Reply:
@vladimir-paramuzov @dmitry-gorokhov we'd like to merge, could you take a look?